lib/stores/auth.ts

```diff
@@ -18,6 +20,7 @@ export interface Sponsorship {
   amount: number
   isActive: boolean
   createdAt: Date
+  hasTeamAccess: boolean;
```

Comment:
Where does this come from?

Reply:
Leftover from a previous strategy, I'll remove it